process-engine/bpmn-studio
# ✨ Add Solution Explorer

## What did you change?

This branch adds the SolutionExplorer to BPMN-Studio!

Waits on:

- [x] https://github.com/process-engine/solutionexplorer.service/pull/2
- [x] https://github.com/process-engine/bpmn-studio/pull/523

Relates to:

https://github.com/process-engine-pro/organization/issues/153

## How can others test the changes?

- Open BPMN-Studio with `npm run electron-start-dev`.
- Open SolutionExplorer.
- Click on Folder to see the FileSystem index tab.
- Click on "Open a Solution".
- Choose a folder with .bpmn files in it.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
